en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -64,354 +60,350 @@ In 2013, game designer Eric Zimmerman published a “Manifesto for a Ludic Centu
+Rather than use complicated scripting to ensure that the year only appears on one side of each printed card, we instead provide instructions to print everything that was explicited designated in the linked Excel spreadsheet. In the spreadsheet, the first 58 rows provide an image, a description, and a year, to be printed on the face of the card. The next 58 rows only provide an image and a description, because the 'year' field is left empty.  
 
-{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="Two pages of cards in a grid format. The left hand page of cards feature years printed in bold and the right hand side of cards are the same but without the year printed." caption="Figure 7: The simplest way to have _nanDECK_ generate a card side without the date being printed is to duplicate the information used to make cards without text in the date field." %}
+{% include figure.html filename="or-en-designing-a-timeline-tabletop-simulator-7.png" alt="A screenshot of part of an Excel spreadsheet with three columns. In column A, the first four lines have a year indicated, but the next three entries do not. The middle column features the description of an event to be printed on the card. The last column lists the file location of the image for the card." caption="Figure 7. The simplest way for nanDECK to generate a card side without the date is to duplicate the information and remove the 'year' field from one half." %}
 
 This is admittedly not the most efficient method, as the information needed to generate new cards must be added twice into the spreadsheet (one entry with the year and one entry without), but it does produce the desired end result.
 
 ### Lines 11 and 12: DUPLEX and PRINT
 
-One of the greatest strengths of _nanDECK_ is its ability to generate synchronized backs and faces of cards. It does this through the DUPLEX and PRINT directives.
+One of the greatest strengths of nanDECK is its ability to generate synchronized faces and backs of cards. It does this through the `DUPLEX` and `PRINT` directives.
 
-> DUPLEX
-> This directive copies a card (or a range of cards) to another position (or range) calculated automatically by the software, it is useful to manage duplicates or synchronize the front face and back of cards for a duplex printing
+> DUPLEX      
+> This directive copies a card (or a range of cards) to another position (or range of positions) calculated automatically by the software. It is useful to manage duplicates, or synchronize the face and back of cards for a duplex printing.
 
-> DUPLEX = “range front”, “range back”, number
+```
+DUPLEX = “range face”, “range back”, number
```

Comment:
I've changed 'front' to 'face' here as well, because I believe "range face" is just a placeholder, and shouldn't affect the code. Do let me know if it is indeed ok this way!